float64 gimbal_yaw_min
float64 gimbal_yaw_max